doc_classes/Ability.xml

```diff
@@ -116,28 +116,29 @@
 			<param index="2" name="ability_container" type="AbilityContainer" />
 			<param index="3" name="runtime_ability" type="RuntimeAbility" />
 			<description>
-				Called when the ability is ticked. This will be called every frame and will replace the [RuntimeAbility] tick method entirely.
-			</description>
+				Called when the ability is ticked.This will be called every frame and will replace the [RuntimeAbility] tick method entirely.[br]
+                [b]Warning:[/b][br]
+                Overriding this method will cause the duration and cooldown handling mechanisms of the ability system to fail, as well as the per-frame `_should_*` method checks.
```

Comment:
I would modify this sentence because those checks are not called if the `_on_tick` is overridden (overriding this virtual simply puts the overrider function in full control on how it ticks)